ext/tag_edit/theme.php

```diff
@@ -69,20 +62,12 @@ public function get_user_editor_html(Image $image): string
         $h_av = $image->get_owner()->get_avatar_html();
         $h_date = autodate($image->posted);
         $h_ip = $user->can(Permissions::VIEW_IP) ? " (".show_ip($image->owner_ip, "Post posted {$image->posted}").")" : "";
-        return "
-			<tr>
-				<th>Uploader</th>
-				<td>
-		".($user->can(Permissions::EDIT_IMAGE_OWNER) ? "
+        return ($user->can(Permissions::EDIT_IMAGE_OWNER) ? "
 					<span class='view'><a class='username' href='".make_link("user/$h_owner")."'>$h_owner</a>$h_ip, $h_date</span>
 					<input class='edit' type='text' name='tag_edit__owner' value='$h_owner'>
 		" : "
 					<a class='username' href='".make_link("user/$h_owner")."'>$h_owner</a>$h_ip, $h_date
-		")."
-				</td>
-				<td width='80px' rowspan='4'>$h_av</td>
-			</tr>
-		";
+		"); //TODO: Implement removed <td width='80px' rowspan='4'>$h_av</td>
```

Comment:
This is the only thing stopping me from merging at the moment, as avatars are a pretty important part of user-identity ^^

I'd be tempted to implement it by adding it to `ext/view/theme.php` - it's not ideal, but it would be a small step back while everything else here is several steps forwards

Reply:
oh, don't worry. I'll implement it accordingly. This PR is not meant to be merged yet as it is a bit less polished, because I'm working on multiple things at the same time. I wanted to make sure I wasn't making a big mistake and have to discard too much work. I'm working on setup's layout at the moment, which was a big pain when I tried to theme it.